cluster/gce/gci/configure-helper.sh

```diff
@@ -328,8 +329,7 @@ kind: Config
 users:
 - name: kubelet
   user:
-    client-certificate-data: ${KUBELET_CERT}
-    client-key-data: ${KUBELET_KEY}
+    token: ${KUBELET_TOKEN}
```

Comment:
I saw it in the token file and the master was using it.  I can revert for getting the rest of this pull.